homeassistant/components/workday/config_flow.py

```diff
@@ -67,7 +68,30 @@ def add_province_to_schema(
         ),
     }
 
-    return vol.Schema({**DATA_SCHEMA_OPT.schema, **add_schema})
+    return vol.Schema({**add_schema, **schema.schema})
+
+
+def add_language_to_schema(
```

Comment:
If a user uses English with "christmas" and then changes to German it will not find it anymore right?

Reply:
That's a possible scenario when German language holidays localization exist for user's country.

I wonder if it's worth mentioning that at [strings.json:33](https://github.com/home-assistant/core/pull/96497/files#diff-ad4c9e983391df7ebc852868dc7ebf2cc4d0f85766b9cb2ec89ec69a7a09507fR33)

Reply:
Yes. I think it should be mentioned and as it's in the same form (right?) it should be pretty visible to the user.
It's also later logging if it can't find a holiday listed so I think that's probably enough what we can do.

Reply:
Hi @arkid15r 
Do you want to pursue this? Otherwise I suggest you might close this one and I'll make a new one. I think this one is a bit too far behind to make good with rebasing to catch up.
Thanks

Reply:
Please take it over if you can. I guess you're more familiar w/ all the HA context quirks. The PH part changes have been included in this PR so it shouldn't be a problem to reuse them. At this moment I'm unable to get this PR to a meaningful conclusion.

Thank you, @gjohansson-ST